src/seedsigner/models/encode_qr.py

```diff
@@ -344,11 +344,13 @@ def derivation_to_keypath(path: str) -> list:
             return Keypath(arr, self.root.my_fingerprint, len(arr))
             
         origin = derivation_to_keypath(self.derivation)
+        self.use_info = None if self.network == "M" else CoinInfo(type=None, network=1)
```

Comment:
An explanatory comment would help here and/or a reference to Issue #582.

Reply:
You are right. Done on commit 0496eea bellow.

Reply:
@fedebuyito I meant in the code itself:

```python
        origin = derivation_to_keypath(self.derivation)

        # For testnet we need to... blah blah... because... blah blah...
        self.use_info = None if self.network == "M" else CoinInfo(type=None, network=1)
```

It's not urgent, but since this PR is already merged, can you add that code comment and submit that change as its own PR?

Reply:
Hi @kdmukai .Yes, sure (sorry for the misunderstanding). Here it is: [df36c1a](https://github.com/SeedSigner/seedsigner/pull/643/commits/df36c1a26a34f9b6e74224d5a6d948663ee2cd2b) .

PR #643